src/dcqc/tests/tests.py

```diff
@@ -158,3 +158,27 @@ def generate_process(self) -> Process:
             command_args=command_args,
         )
         return process
+
+
+class TiffTag306DateTimeTest(ExternalTestMixin, TestABC):
+    tier = 4
+
+    def generate_process(self) -> Process:
+        file = self.get_file()
+        path = file.local_path.as_posix()
+        command_args = [
+            "!",  # negate exit status
```

Comment:
So for whatever reason, this `!` is getting passed to the command that is executed in Nextflow as '!'. Resulting in an error. Example:
```
'!' tifftools dump /Users/bmacdonald/Development/repos/nf-dcqc/work/ee/a6706f3685c8410c0194755dfc2223/dcqc-staged-4xemb67_/single-channel.tif '|' grep -a -q ''\"'\"'DateTime 306 (0x132) ASCII'\"'\"''"
```

Reply:
After further investigation, this is happening because of the behavior of `shlex.join()` in [`test_abc.py`](https://github.com/Sage-Bionetworks-Workflows/py-dcqc/blob/c4f684cda358acac5be9edff006931291e88f21d/src/dcqc/tests/test_abc.py#L188).

It escapes/quotes all special characters when joining command arguments into one "shell-escaped" command string. Just to see, I replaced the `shlex.join()` with `" ".join()` which resulted in this command:
```
! tifftools dump /Users/bmacdonald/Development/repos/nf-dcqc/work/ee/a6706f3685c8410c0194755dfc2223/dcqc-staged-4xemb67_/single-channel.tif '|' grep -a -q ''\"'\"'DateTime 306 (0x132) ASCII'\"'\"''"
```
 and the tests passed.

Reply:
Awesome  - feel free to push your changes to this branch.  

Interesting, did you also have a change to run it on the file thats supposed to fail?